README.md

```diff
@@ -973,9 +974,9 @@ _concat([1, 2], 'buzz');
 
 The type of `_concat` is misleading: it suggests that it can operate on any
 two values of *any* one type. In fact there's an implicit constraint, since
-the type must support concatenation (in [mathematical][10] terms, the type
-must have a [semigroup][11]). The run-time type errors that result when this
-constraint is violated are not particularly descriptive:
+the type must support concatenation (in [mathematical][semigroup] terms, the
+type must have a [semigroup][Semigroup]). The run-time type errors that result
+when this constraint is violated are not particularly descriptive:
```

Comment:
Link identifiers are *not* case sensitive. I changed `[Semigroup]` to `[FL:Semigroup]`. I also wrote a program to verify that I didn't make any typos when updating the identifiers:

```console
$ for branch in master dc-link-identifiers ; do git checkout $branch && pandoc README.md | shasum -a 256 ; done
Switched to branch 'master'
Your branch is up-to-date with 'origin/master'.
ee3aaab7734c9e7380fbe8eb881bc61c9c14c33c8d8a485cfc937edf50ad486a  -
Switched to branch 'dc-link-identifiers'
ee3aaab7734c9e7380fbe8eb881bc61c9c14c33c8d8a485cfc937edf50ad486a  -
```